### PR TITLE
CXF-91821 Added Oracle Test to PROD GHA

### PR DESCRIPTION
**Pull request by jkallem-equinix**
Updated Prod GHA workflow and function by adding Oracle test case in below files
**.github/workflows/terratests-prod-suite.yaml
tests/prod/prod_sanity_suite_test.go**

Added FCR2Port Connection_name and bandwidth Upgrade in 
**tests/prod/prod_sanity_suite_test.go**

As Discussed, I have Updated Source Paths to local Paths for below resources:
**cloud-router-2-aws-connection
cloud-router-2-azure-connection
cloud-router-2-port-routing-protocol-connection
port-2-aws-connection
port-2-azure-connection
port-2-wan-connection
virtual-device-2-azure-connection
virtual-device-2-port-connection
virtual-device-2-wan-connection**

### CODE REVIEW

File: tests/examples-without-external-providers/cloud-router-2-port-routing-protocol-connection/main.tf [hunk 1]
```diff
@@ -1,10 +1,11 @@
 provider "equinix" {
   client_id     = var.equinix_client_id
   client_secret = var.equinix_client_secret
+  endpoint = "https://uatapi.equinix.com"
```
Comment (thogarty) [inline tests/examples-without-external-providers/cloud-router-2-port-routing-protocol-connection/main.tf:4]: Should not include UAT URL in main.tf files.

**Review by thogarty — CHANGES_REQUESTED** [review]
One small change requested. Rest is ok.

File: tests/examples-without-external-providers/cloud-router-2-port-routing-protocol-connection/main.tf (hunk 1 above)
Comment (jkallem-equinix) [reply tests/examples-without-external-providers/cloud-router-2-port-routing-protocol-connection/main.tf]: Removed it, Thanks a lot.

File: .github/workflows/documentation.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
     name: TF docs
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4
```
Comment (thogarty) [inline .github/workflows/documentation.yaml:18]: This is overwritting another change. We need to not have any update in your PR here. Please ensure this is not changed in your PR.

**Review by thogarty — CHANGES_REQUESTED** [review]
Another change requested.

**Review by thogarty — APPROVED** [review]
LGTM. Approved.

**Pull request merged**